client/app/cloud/project/compute/infrastructure/translations/Messages_fr_FR.json

```diff
@@ -272,6 +272,10 @@
   "cpci_vm_flavor_category_g3-240": "G3-240",
   "cpci_vm_flavor_category_t1-45": "T1-45",
   "cpci_vm_flavor_category_t1-90": "T1-90",
+  "cpci_vm_flavor_category_t1-180": "T1-180",
```

Comment:
It's a nonsense to provide these references to the translations team.

Since we already have references in this file, OK, but do you find any pluvalue to add them into a `Messages_xx_xx.json` file?

Reply:
These should be constants

Reply:
Yes, I was thinking the same. Its no use to add these in the translation file. I have added because it was already done in this way. What should we do?

Reply:
Well the naming is always `${_.uppercase(firstPart)}-${secondPart}`, isn't it ? Is there any trap ?